tests_uploadprogress.js

```diff
@@ -0,0 +1,15 @@
+const
+  Oyster = require(`./dist/index.js`).default,
+  IOTA = require(`iota.lib.js`)
+
+const iota = new IOTA({ provider: 'https://poll.oysternodes.com:14265/' })
+
+const progress = new Oyster.UploadProgress(`coniccss1b19c83c8fbdc10476d6803e82217061e903197819353fd5885c8b29cf599e11V6b2AUBh`, {
+  iotaProvider: iota
```

Comment:
It fails without an IOTA instance when I tested it. I believe the call that it needs is `iota.api.findTransactions`. I think we should revise the way IOTA instances are handled so that only the provider string is required to be passed. See: https://oysterprotocol.slack.com/archives/C8R52D5PF/p1534219434000100

Reply:
Oh ok. Yes, we shouldn't have the caller pass in an IOTA instance. We can change that in a future PR though.

IDK if this library is already being used in by the community already, but that would be a breaking change. We would need to update the package number to a major update.